cmd/start.go

```diff
@@ -7,37 +7,39 @@ import (
 	"net/http"
 	"os"
 	"os/signal"
+	"strings"
 	"syscall"
 	"time"
 
-	"github.com/robfig/cron"
-
-	log "github.com/sirupsen/logrus"
-
 	"github.com/open-feature/flagd/pkg/eval"
 	"github.com/open-feature/flagd/pkg/runtime"
 	"github.com/open-feature/flagd/pkg/service"
 	"github.com/open-feature/flagd/pkg/sync"
+	"github.com/robfig/cron"
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
-var (
-	serviceProvider   string
-	syncProvider      string
-	evaluator         string
-	uri               []string
-	servicePort       int32
-	socketServicePath string
-	bearerToken       string
-	serverCertPath    string
-	serverKeyPath     string
+const (
+	portFlagName            = "port"
+	serviceProviderFlagName = "service-provider"
+	socketPathFlagName      = "socketpath"
+	syncProviderFlagName    = "sync-provider"
+	evaluatorFlagName       = "evaluator"
+	serverCertPathFlagName  = "server-cert-path"
+	serverKeyPathFlagName   = "server-key-path"
+	uriFlagName             = "uri"
+	bearerTokenFlagName     = "bearer-token"
 )
 
 func findService(name string) (service.IService, error) {
+	serverCertPath := viper.GetString(serverCertPathFlagName)
+	serverKeyPath := viper.GetString(serverKeyPathFlagName)
 	registeredServices := map[string]service.IService{
 		"http": &service.HTTPService{
 			HTTPServiceConfiguration: &service.HTTPServiceConfiguration{
-				Port:           servicePort,
+				Port:           viper.GetInt32(portFlagName),
```

Comment:
Forced to go for this `viper` retrieval approach: https://github.com/spf13/viper/issues/85